### ISSUE TITLE
The amount of data generated by prepare_data_dwn is not the same

### ISSUE THREAD

**Issue by stanny880913** [longyunf/radiant#15]
Hello, I trained the structure of your paper twice and ran all the processes completely. However, when I did the `--do_eval` of train_dwn again, I found that the amount of data was different. I used the ones generated through prepare_data_dwn. Val.json is used for verification. Is this normal? Or could something possibly go wrong?
thank you~

**Comment by longyunf** [comment]
You can check if the arguments of dataloader are the same.
https://github.com/longyunf/radiant/blob/cf5355396d42ef17940e29ef8f9e3cabfd8035c3/scripts/train_dwn.py#L286-L289

**Comment by stanny880913** [comment]
> You can check if the arguments of dataloader are the same.
> 
> https://github.com/longyunf/radiant/blob/cf5355396d42ef17940e29ef8f9e3cabfd8035c3/scripts/train_dwn.py#L286-L289

Hello, my arguments of dataloader are all the same, Will the model accuracy of radar branch affect the amount of data?
thank you

**Comment by longyunf** [comment]
The accuracy of radar branch may influence radar-camera association results.

**Comment by longyunf** [comment]
You can also check if val.json changes.

**Comment by stanny880913** [comment]
> The accuracy of radar branch may influence radar-camera association results.

Hello, so the higher the accuracy of the radar branch model, the more data will be generated by `prepare_dwn_data`(include train.json and val.json)?

**Comment by stanny880913** [comment]
> The accuracy of radar branch may influence radar-camera association results.

How can I check radar branch model accuracy, Is it output through `error_rd` of `--do_eval` in `train_dwn.py`? Determine the quality of the radar branch model based on the loss value?
thank you
